specification/validated-relationships-service-api.yaml

```diff
@@ -481,36 +753,215 @@ components:
+                              enum: ["patient_name_given"]
+                            text:
+                              type: string
+                              example: "Given names, including any middle names."
+                            answer:
+                              type: array
+                              items:
+                                oneOf:
+                                  - type: object
+                                    properties:
+                                      valueString:
+                                        type: array
+                                        description: |
+                                          Each name(s) should be a separate item in the list. The first given name may include multiple names, separated by a space.
+                                          Subsequent names must be broken down into list items. For example, the input `[Jane Marie Anne, Jo Adele]` returns `[Jane Marie Anne, Jo, Adele]`.
+
+                                        maxItems: 5                                  
+                                        example: [Jane Marie Anne]
+                                        items:
+                                          type: string
+                                          maxLength: 35
+                                          example: Jane
+
+                        - type: object
+                          properties:
+                            linkId:
+                              type: string
+                              enum: ["patient_name_family"]
+                            text:
+                              type: string
+                              example: "Family name (often called Surname)."
+                            answer:
+                              type: array
+                              items:
+                                oneOf:
+                                  - type: object
+                                    properties:
+                                      valueString:
+                                        type: string
+                                        example: "Smith"
+
+              - type: object
+                properties:
+                  linkId:
+                    type: string
+                    enum: ["patient_birthDate"]
+                  text:
+                    type: string
+                    example: "Date of Birth"
+                  answer:
+                    type: array
+                    items:
+                      oneOf:
+                        - type: object
+                          properties:
+                            valueString:
+                              type: string
+                              example: "2020-10-22"
+
+    QuestionnaireItem_RequestedAccess:
+      type: object
+      properties:
+        linkId:
+          type: string
+          enum: ["requestedAccess"]
+        text:
+          type: string
```

Comment:
Suggest adding an example "Details about request access" for parity with most other fields, currently this one comes out as `"text": "string"` in the generated swagger docs.